currentpaymentversion.civix.php

```diff
@@ -6,10 +6,10 @@
  * The ExtensionUtil class provides small stubs for accessing resources of this
  * extension.
  */
-class CRM_Currentpaymentversion_ExtensionUtil {
-  const SHORT_NAME = "currentpaymentversion";
-  const LONG_NAME = "biz.jmaconsulting.currentpaymentversion";
-  const CLASS_PREFIX = "CRM_Currentpaymentversion";
+class CRM_CurrentPaymentVersion_ExtensionUtil {
```

Comment:
Is this and the ClassPrefix going to cause issues here @monishdeb ?

Reply:
yeah I checked earlier in the codebase, apart from civix file it is not used anywhere else: 
```
Monishs-MacBook-Pro:biz.jmaconsulting.currentpaymentversion monish$ grep -irn "CRM_CurrentpaymentVersion_ExtensionUtil" .
./currentpaymentversion.php:4:use CRM_Currentpaymentversion_ExtensionUtil as E;
grep: warning: ./biz.jmaconsulting.currentpaymentversion: recursive directory loop
./currentpaymentversion.civix.php:9:class CRM_CurrentPaymentVersion_ExtensionUtil {
./currentpaymentversion.civix.php:80:use CRM_CurrentPaymentVersion_ExtensionUtil as E;
```
So I don't it will cause any issue.